### ISSUE TITLE
Getting a "Leave site?" pop-up when adding a phone number field with country code.

### ISSUE THREAD

**Issue by bartdigitalpulse** [verbb/formie#720]
**Description**

Getting a "Leave site? Changes that  you made may not be saved." pop-up when adding a phone number field with country code.

![Screenshot 2022-01-19 at 14 11 35](https://user-images.githubusercontent.com/75615033/150138362-f4ed7e8b-2553-4f1f-856f-05fcdabecc71.png)
![Screenshot 2022-01-19 at 14 10 42](https://user-images.githubusercontent.com/75615033/150138367-e98fecc2-ff3a-43aa-8a9d-392f691d5fef.png)

**Steps to reproduce**
1. Add phone number field with country code
2. Click link on the page that takes you away from page

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Both
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.5.6
- Craft version: Pro 3.7.28
- Multi-site: Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.6",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.7](https://github.com/verbb/formie/releases/tag/1.5.7)
